packages/wonder-blocks-accordion/src/components/accordion.tsx

```diff
@@ -60,6 +60,15 @@ type Props = AriaProps & {
      * value is prioritized.
      */
     cornerKind?: AccordionCornerKindType;
+    /**
+     * Whether to include animation on the header. This should be false
+     * if the user has `prefers-reduced-motion` opted in. Defaults to false.
+     *
+     * If this prop is specified both here in the Accordion and within
+     * a child AccordionSection component, the Accordion’s includeAnimation
+     * value is prioritized.
+     */
+    includeAnimation?: boolean;
```

Comment:
**suggestion (non-blocking):** I think `animated?: boolean` might fit better with existing boolean props in other components such as `opened`.

Reply:
That's a good point @kevinbarabash, I was thinking about using something like this as well.